runtime/onert/backend/cpu/ops/ConvolutionLayer.cc

```diff
@@ -197,6 +227,15 @@ void ConvolutionLayer::prepare()
     kernel.prepareQuant(getShape(_input), getShape(_kernel), getShape(_output), _strideWidth,
                         _strideHeight, _dilationWidthFactor, _dilationHeightFactor);
   }
+  else if (_input->data_type() == OperandType::QUANT_INT8_ASYMM && _kernel->is_constant() &&
+           !_input->is_dynamic() && !_output->is_dynamic())
+  {
+    kernel.getQuantizedConvolutionMultipliersAndShifts(
+      _input->data_scale(), _output->data_scale(), _kernel->data_scales().data(),
+      _kernel->data_scales().size(), getShape(_kernel).Dims(0));
+    kernel.prepareQuant(getShape(_input), getShape(_kernel), getShape(_output), _strideWidth,
```

Comment:
This function is only necessary when kernel uses `im2col`. It looks like reference kernel doesn't use `im2col`.